[2022-02-15 03:15:45,366] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.mart_orders manual__2022-02-15T03:14:06.780838+00:00 [queued]>
[2022-02-15 03:15:45,379] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.mart_orders manual__2022-02-15T03:14:06.780838+00:00 [queued]>
[2022-02-15 03:15:45,379] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:15:45,380] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 03:15:45,381] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:15:45,397] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): model.my_new_project.mart_orders> on 2022-02-15 03:14:06.780838+00:00
[2022-02-15 03:15:45,406] {standard_task_runner.py:52} INFO - Started process 1465 to run task
[2022-02-15 03:15:45,421] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'model.my_new_project.mart_orders', 'manual__2022-02-15T03:14:06.780838+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmp1dz93pw7', '--error-file', '/tmp/tmpatk4rgk7']
[2022-02-15 03:15:45,423] {standard_task_runner.py:77} INFO - Job 119: Subtask model.my_new_project.mart_orders
[2022-02-15 03:15:45,498] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.model.my_new_project.mart_orders manual__2022-02-15T03:14:06.780838+00:00 [running]> on host 459d09dd7c86
[2022-02-15 03:15:45,592] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=model.my_new_project.mart_orders
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T03:14:06.780838+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T03:14:06.780838+00:00
[2022-02-15 03:15:45,622] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 03:15:45,623] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json run --target dev --models mart_orders             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 03:15:45,645] {subprocess.py:85} INFO - Output:
[2022-02-15 03:15:51,412] {subprocess.py:89} INFO - 03:15:51  Running with dbt=1.0.1
[2022-02-15 03:15:51,738] {subprocess.py:89} INFO - 03:15:51  Found 8 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 03:15:51,745] {subprocess.py:89} INFO - 03:15:51
[2022-02-15 03:15:55,229] {subprocess.py:89} INFO - 03:15:55  Concurrency: 4 threads (target='dev')
[2022-02-15 03:15:55,232] {subprocess.py:89} INFO - 03:15:55
[2022-02-15 03:15:55,259] {subprocess.py:89} INFO - 03:15:55  1 of 1 START view model dbt_bytecode_training.mart_orders....................... [RUN]
[2022-02-15 03:15:57,460] {subprocess.py:89} INFO - 03:15:57  1 of 1 OK created view model dbt_bytecode_training.mart_orders.................. [SUCCESS 1 in 2.20s]
[2022-02-15 03:15:57,470] {subprocess.py:89} INFO - 03:15:57
[2022-02-15 03:15:57,472] {subprocess.py:89} INFO - 03:15:57  Finished running 1 view model in 5.72s.
[2022-02-15 03:15:57,476] {subprocess.py:89} INFO - 03:15:57
[2022-02-15 03:15:57,479] {subprocess.py:89} INFO - 03:15:57  Completed successfully
[2022-02-15 03:15:57,481] {subprocess.py:89} INFO - 03:15:57
[2022-02-15 03:15:57,483] {subprocess.py:89} INFO - 03:15:57  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 03:15:57,877] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 03:15:57,909] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=model.my_new_project.mart_orders, execution_date=20220215T031406, start_date=20220215T031545, end_date=20220215T031557
[2022-02-15 03:15:57,968] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 03:15:58,003] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
